DevEdu.API/DevEdu.Business.Tests/TestDataHelpers/NotificationData.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.Collections.Generic;
+using DevEdu.DAL.Models;
+
+namespace DevEdu.Business.Tests
+{
+    public static class NotificationData
+    {
+        public const int ExpectedNotificationId = 42;
+        public const int NotificationId = 1;
+        public const int UserId = 1;
+        public const int GroupId = 1;
```

Comment:
it's a bad practice to have such values as consts
it's better to declare them in each test method separately